Cards/Effects/EffectDamage.gd

```diff
@@ -0,0 +1,12 @@
+class_name EffectDamage extends EffectBase
+
+# @Override
+func apply_effect(caster: Entity, target: Entity, value: int) -> void:
+    var target_damage_data: DealDamageData = DealDamageData.new()
+    target_damage_data.damage = value
+    target_damage_data.caster = caster
+    target.get_health_component().deal_damage(target_damage_data)
+
+# TODO : what to do for Status that inflige damage over time ?
+    # Should they call an EffectDamage when they trigger each turn ?
+    # or should they directly call deal_damage on the target ?
```

Comment:
Could do either, but makes more sense to just call deal_damage I think

Reply:
I agree

Reply:
Will probably open a new issue for the Status modifications I think.

Reply:
I realized Status already inflict damage by calling deal_damage, so no need for an issue